web/src/components/Plugin/UI/limit-count.tsx

```diff
@@ -135,7 +145,7 @@ const RedisClusterForm: React.FC = () => {
                       </Form.Item>
                     </Col>
                     <Col style={{ ...removeBtnStyle, marginLeft: -10 }}>
-                      {fields.length > 1 ? (
+                      {fields.length > minLength ? (
```

Comment:
I think it would be better to change this part of the Conditional (ternary) operator to the && operator